sqldelight-compiler/integration-tests/src/test/kotlin/com/example/TeamQueries.kt

```diff
@@ -52,7 +52,7 @@ class TeamQueries(private val database: TestDatabase, private val driver: SqlDri
 
     private inner class TeamForCoach<out T : Any>(private val coach: String, mapper: (SqlCursor) ->
             T) : Query<T>(teamForCoach, mapper) {
-        override fun execute(): SqlCursor = driver.executeQuery(80, """
+        override fun execute(): SqlCursor = driver.executeQuery(0, """
```

Comment:
can you do `./gradlew clean build` from the command line and then commit these files. Its an annoying work flow but it will give the correct results for these identifiers

Reply:
Done

Reply:
Hmm, but it didn't seem to work. The identifiers are still changed. But perhaps that's to be expected?

Reply:
thats fine, as long as its from a clean build then building master will create the same ids